test/unit/membership_test.js

```diff
@@ -266,7 +266,7 @@ testRingpop('set adds all members', function t(deps, assert) {
 });
 
 testRingpop('set emits an event', function t(deps, assert) {
```

Comment:
The name of this test doesn't cover what it is doing anymore, we should rename it, or duplicate it to only test the additional call to the `handleUpdate` function.